pkg/plugins/k8s/k8s_plugin.go

```diff
@@ -154,6 +154,11 @@ func (p *K8sPlugin) OnNodeStateChange(new *sriovnetworkv1.SriovNetworkNodeState)
 	return
 }
 
+// OnNodeStatusChange verify whether SriovNetworkNodeState CR status present changes on configured VFs.
+func (p *K8sPlugin) CheckStatusChanges(*sriovnetworkv1.SriovNetworkNodeState) (bool, error) {
+	return false, nil
```

Comment:
Hi please a TODO and comment here that we skip the externallyManage with a link to the issue we open on the repo please

Reply:
the problem is with the method NeedToUpdateSriov which is not used by this plugin